src/bin/signal_handler.rs

```diff
@@ -65,7 +65,7 @@ mod imp {
         for sig in trap {
             match sig {
                 SIGTERM | SIGINT | SIGHUP => {
-                    info!("receive signal {}, stopping server...", sig);
+                    info!("receive signal {}, stopping server...", sig as c_int);
```

Comment:
Why cast is required here?

Reply:
This is required because of the signal version upgrade from 1.2 to 1.4 and the changes that went into it to upgrade nix